### ISSUE TITLE
SohoWizard: wizard in modal throws exception to console

### ISSUE THREAD

**Issue by guoliang** [infor-design/enterprise-ng#1769]
**Describe the bug**

With class name `modal-buttonset` like the one below, it cause error to be thrown in console
```html
<div
  class="modal-buttonset"
  soho-wizard-buttonbar
  [buttons]="buttons"></div>
```
![image](https://github.com/user-attachments/assets/ae225a87-8039-43e8-9740-6945d75ab266)

If I remove the class name, it works, but the desired styling is not there.

We want the wizard buttons to look as following
![image](https://github.com/user-attachments/assets/5681c726-9b8a-4ab3-8a5e-ee6c47560f96)

Not as following
![image](https://github.com/user-attachments/assets/4a045165-ba0e-47c2-b26c-47333be201a9)

**To Reproduce**
1. Create a minimal example where `SohoWizard` is used in `SohoModal`

**Expected behavior**
No error in console

**Version**
- ids-enterprise-ng: 18.2.2

**Comment by tjamesallen15** [comment]
Hi @guoliang ,

cc @tmcconechy @ericangeles 

I added what's in the description in our example page but there are no errors on the browser. Perhaps the steps to reproduce is incomplete? Can you update us? See below for reference.

http://localhost:4200/ids-enterprise-ng-demo/wizard

![image](https://github.com/user-attachments/assets/bd4da8e4-8c74-46dd-93c8-a9d641243b02)

![image](https://github.com/user-attachments/assets/53bb8e03-d9f1-407f-a6f4-e7819dde523b)

**Comment by guoliang** [comment]
I think it may be for a specific setup in which the error would appear. I'll see if I get time to make a small example later this week.

**Comment by tmcconechy** [comment]
@tjamesallen15 yeah i dont know either. If @guoliang will make an example and we can skip for now and come back later
